packages/terra-application-name/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated `waitForDisplayed` to `waitForExist` to fix failing tests in themes.
```

Comment:
I believe this issue was occurring only in clinical theme. so the log can be updated as below.

```suggestion
  * Updated `waitForDisplayed` to `waitForExist` to fix failing tests in cerner clinical theme.
```